package.json

```diff
@@ -24,7 +24,8 @@
   ],
   "scripts": {
     "build": "rm -rf dist && rollup -c --configPlugin @rollup/plugin-typescript",
-    "u": "pnpm update --latest !eslint !react !react-dom !@types/react !@types/react-dom"
+    "u": "pnpm update --latest !eslint !react !react-dom !@types/react !@types/react-dom",
+    "prepare": "npm run build"
```

Comment:
Sorry, why do we need this script?

Reply:
We need this script because the original module is being published to npm (?), and it's automatically built before we import it.
Since we are not publishing this forked module, we need to add this script so it's actually built.

Without this script, the imported module doesn't contain the `dist` folder and is not usable.

Here is the doc for reference:
https://docs.npmjs.com/cli/v8/using-npm/scripts#life-cycle-scripts

>NOTE: If a package being installed through git contains a prepare script, its dependencies and devDependencies will be installed, and the prepare script will be run, before the package is packaged and installed.

Reply:
got it, thank you.